frontend/app/templates/application.hbs

```diff
@@ -4,7 +4,7 @@
 {{else if (not-eq this.route "embed")}}
   <HeadLayout/>
   <MainHeader @isTeach={{false}} @logout={{this.logout}} />
-  <div class="container-fluid m-0 p-0 ">
+  <div class="container">
```

Comment:
this will cause the bar below the main navigation menu (at the homepage) to have some left & right margins.

Is this deliberate? 

![Screenshot from 2021-04-21 20-39-50](https://user-images.githubusercontent.com/25152892/115597253-bef4fd00-a2e1-11eb-974e-a0a0a649a17d.png)

Reply:
This was intentional